src/core/ext/xds/xds_channel_creds.cc

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright 2019 gRPC authors.
+// Copyright 2022 gRPC authors.
```

Comment:
no need to update copyright years

Reply:
This is a new file I added in the previous PR and I forgot to put 2022 after copied and fixed.
Do you still want to keep it as 2019?

Reply:
Closing the comment after chatting off-line. Will merge on Monday.